.pipelines/dwsql-pipelines.yml

```diff
@@ -0,0 +1,220 @@
+    inputs:
+      script: 'echo ##vso[task.setvariable variable=publishverify]Yes'
+
+  - task: NuGetAuthenticate@0
+    displayName: 'NuGet Authenticate'
+
+  - task: UseDotNet@2
+    displayName: Setup .NET SDK v6.0.x
+    inputs:
+      packageType: sdk
+      version: 6.0.x
+
+  - task: NuGetToolInstaller@1
+
+  - task: NuGetCommand@2
+    displayName: Restore NuGet packages
+    inputs:
+      restoreSolution: '$(solution)'
+      feedsToUse: config
+      nugetConfigPath: Nuget.config
+
+  - task: PowerShell@2
+    displayName: Install SQL LocalDB # Update when clarity on how to setup 
+    inputs:
+      targetType: 'inline'
+      script: |
+        SqlLocalDb.exe start
+        SqlLocalDB.exe info "MSSQLLocalDB"
+        Write-Host "Downloading"
+        Import-Module BitsTransfer
+        Start-BitsTransfer -Source $(InstallerUrl) -Destination SqlLocalDB.msi
+        Write-Host "Installing"
+        Start-Process -FilePath "SqlLocalDB.msi" -Wait -ArgumentList "/qn", "/norestart", "/l*v SqlLocalDBInstall.log", "IACCEPTSQLLOCALDBLICENSETERMS=YES";
+        SqlLocalDB.exe stop MSSQLLocalDB -k
+        SqlLocalDB.exe delete MSSQLLocalDB
+
+  - task: PowerShell@2
+    displayName: 'Start MSSQLLocalDB'
+    inputs:
+      targetType: 'inline'
+      script: |
+        SqlLocalDb.exe start MSSQLLocalDB
+        SqlLocalDb.exe info "MSSQLLocalDB"
+
+  - task: DotNetCoreCLI@2
+    displayName: Build
+    inputs:
+      command: build
+      projects: '**/*.csproj'
+      arguments: '-p:generateConfigFileForDbType=DwSql --configuration $(buildConfiguration)' # Update this to match your need
+    
+  - task: FileTransform@1.206.0
+    displayName: 'Generate dab-config.DwSql.json'
+    inputs:
+      folderPath: '$(System.DefaultWorkingDirectory)'
+      fileType: 'json'
+      targetFiles: 'src/out/tests/net6.0/dab-config.DwSql.json'
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Run DwSql Integration Tests'
+    inputs:
+      command: test
+      arguments: '--filter "TestCategory=DwSql" --no-build --configuration $(buildConfiguration) --collect "XPlat Code coverage"'
+      projects: '**/*Tests/*.csproj'
+
+  - task: PublishCodeCoverageResults@1
+    displayName: 'Publish code coverage'
+    inputs:
+      codeCoverageTool: Cobertura
+      summaryFileLocation: '$(Agent.TempDirectory)/**/*cobertura.xml'
+
+  - task: CopyFiles@2
+    condition: eq(variables['publishverify'], 'Yes')
+    displayName: 'Copy received files to Artifact Staging'
+    inputs:
+      contents: '**\*.received.*' 
+      targetFolder: '$(Build.ArtifactStagingDirectory)\Verify'
+      cleanTargetFolder: true
+      overWrite: true
+
+  - task: PublishBuildArtifacts@1
+    displayName: 'Publish received files as Artifacts'
+    name: 'verifypublish'
+    condition: eq(variables['publishverify'], 'Yes')
+    inputs:
+      PathtoPublish: '$(Build.ArtifactStagingDirectory)\Verify'
+      ArtifactName: 'Verify'
+      publishLocation: 'Container'
+      
+ 
```

Comment:
For some reason, this file still shows that its missing a new line at the end of file:

![image](https://github.com/Azure/data-api-builder/assets/3513779/89552157-22ce-4703-93a4-25d16de7744e)

Reply:
Added new line in pr again. Going to resolve this for now to unblock. Will take a look at it in the query building pr where ill actually use this pipeline.